src/components/Header/Header.tsx

```diff
@@ -9,6 +9,8 @@ const StyledHeader = styled.header`
   align-items: stretch;
   background-color: var(--color-blue);
   flex: 0 0 56px;
+  box-shadow: 0px 2px 10px 0px rgba(7, 77, 118, 0.2);
+  z-index: 999;
```

Comment:
OK yes can agree to that system. However I'd also suggest using 90 as the topmost layer (safe to say we won't have more than 10 layers for this site), as otherwise some arbitrary 30 doesn't strike immediately as being a top one.

Reply:
absolutely, I would agree with any system as long as it's manageable